<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1076</width>
    <height>744</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="StockOperationsTabs">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>991</width>
     <height>531</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="StockGroupTab">
    <attribute name="title">
     <string>Stock Group</string>
    </attribute>
    <widget class="QFrame" name="StockGroupFrame">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>40</y>
       <width>611</width>
       <height>371</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="HeadingLabel">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>491</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Create a New Stock Group&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="CreateStockGroupButton">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>250</y>
        <width>501</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Create Stock Group</string>
      </property>
     </widget>
     <widget class="QLabel" name="DescLabel">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>190</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Description</string>
      </property>
     </widget>
     <widget class="QLabel" name="NameLabel">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>120</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Group Name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="GroupNameText">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>113</y>
        <width>331</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="GroupDescText">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>180</y>
        <width>331</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="GroupWarningText">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>320</y>
        <width>371</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="StockTab">
    <attribute name="title">
     <string>New Stock</string>
    </attribute>
    <widget class="QFrame" name="StockFrame">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>40</y>
       <width>611</width>
       <height>421</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="StockHeadingLabel">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>491</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Create a New Stock&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="CreateStockButton">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>320</y>
        <width>501</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Create Stock</string>
      </property>
     </widget>
     <widget class="QLabel" name="StockDescText_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>140</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Description</string>
      </property>
     </widget>
     <widget class="QLabel" name="StockNameLabel">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>80</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Stock Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="MiniCountLabel">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>270</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Minimum Count</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="StockMiniCount">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>260</y>
        <width>331</width>
        <height>41</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>10000</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="StockNameText">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>80</y>
        <width>331</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="StockDescText">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>140</y>
        <width>331</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="StockWarningText">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>370</y>
        <width>451</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QComboBox" name="StockGroupBox">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>200</y>
        <width>331</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="GroupSelectLabel">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>200</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Stock Group</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="MoreActions">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>120</y>
     <width>451</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>More Actions</string>
   </property>
   <widget class="QCommandLinkButton" name="BackToStockList">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Back To Stock List Pages</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>40</y>
     <width>561</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; color:#613583;&quot;&gt;Stock Group and Stock Operations&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
